team.md

```diff
@@ -8,5 +8,4 @@
 
 
 **Zachary Garza**
-
-
+I'm a student with an afro that likes to play guitar hero and hang out with friends.
```

Comment:
Guitar Hero should be capitalized because it is the name of a game.

Reply:
Can you specify which guitar it is? (electric or acoustic) thanks!